src/main/java/com/microsoft/azure/vmagent/AzureVMAgentCleanUpTask.java

```diff
@@ -432,7 +439,7 @@ public Void call() throws Exception {
     }
 
     public AzureVMCloud getCloud(final String cloudName) {
-        return Jenkins.getInstance() == null ? null : (AzureVMCloud) Jenkins.getInstance().getCloud(cloudName);
+        return (AzureVMCloud) Jenkins.getInstance().getCloud(cloudName);
```

Comment:
There is an annotation `CheckForNull` for `Jenkins.getInstance`. Which means you have to check `null` before `getCloud` otherwise UT/IT will fail.